Documentation/ceph-osd-mgmt.md

```diff
@@ -63,6 +63,8 @@ through the removal process:
 If all the PGs are `active+clean` and there are no warnings about being low on space, this means the data is fully replicated
 and it is safe to proceed. If an OSD is failing, the PGs will not be perfectly clean and you will need to proceed anyway.
 
+Part of OSD removal can be automated by [rook-ceph-purge-osd job](/cluster/examples/kubernetes/ceph/osd-purge.yaml). Please use it if necessary.
+
 ### From the Toolbox
```

Comment:
What if we replaced this section with the recommendation to run the osd purge job?

Reply:
Sounds good. How about recommend it only for `out` osds? I'm afraid the massive amount of backfilling after purging `down` and `in` OSDs.

Reply:
If an OSD attempts to be purged, the job will fail if it is not down, so at least the job should not purge healthy OSDs. But anyway, it would be good to add a warning that they should be very careful when removing OSDs or else they could lose data.

Reply:
Update this PR to recommend osd purge job and refactor the osd removal process.